src/auth/downloader.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <QObject>
+
+QT_BEGIN_NAMESPACE
+class QNetworkAccessManager;
+class QNetworkReply;
+class QUrl;
+QT_END_NAMESPACE
+
+class Downloader : public QObject
+{
+    Q_OBJECT
+
+private:
+    QNetworkAccessManager *mNetworkAccessManager;
+
+public:
+    Downloader(QObject* pobj = nullptr);
+
+    void downloadGET( const QUrl &url,                const QString &userName,

```

Comment:
you move ` **const QString &tokenOrPassword = "")**` where that was not necessary, but left
`void respond       (const QUrl &url, const QMap<QString, QString> &headersMap, const QMap<QString, QString> &respondMap);` in one line.
The same for the fucntion 
`void DownloaderWrapper::onRespond(const QUrl &url, const QMap<QString, QString> &headersMap, const QMap<QString, QString> &respondMap)` in the downloaderWrapper.cpp

Reply:
made in one style